### ISSUE TITLE
Incorrect window

### ISSUE THREAD

**Issue by DimastyK** [rehsd/Emul816or#1]
Hello. After launching emulator window looks Incorrect. Win10 x64
![2023-11-13_184739](https://github.com/rehsd/Emul816or/assets/135803967/c64ec708-3f96-4412-9477-dc4c8b526c17)
![2023-11-13_184752](https://github.com/rehsd/Emul816or/assets/135803967/adce8d03-2ca4-40f6-8769-1db5ac97d1dc)

**Comment by rehsd** [comment]
I’ll take a look at that code. What resolution are you running? I can test it on a few systems, too.

**Comment by rehsd** [comment]
Also, what scaling are you using for your display settings?

**Comment by DimastyK** [comment]
> Also, what scaling are you using for your display settings? 
1280x720 on laptop

**Comment by rehsd** [comment]
@DimastyK, with 1280x720, I assume you are running 100% scaling. If that's not the case, please let me know. I will test the app at a few different resolutions / dpis / scaling. The form doesn't have much intelligence in it currently do deal with different dpi and scaling settings.

**Comment by DimastyK** [comment]
> @DimastyK, with 1280x720, I assume you are running 100% scaling. If that's not the case, please let me know. I will test the app at a few different resolutions / dpis / scaling. The form doesn't have much intelligence in it currently do deal with different dpi and scaling settings.

yes, 100% scaling

**Comment by rehsd** [comment]
@DimastyK, I was able to reproduce the issue. I have the form setup for a higher resolution. I'll make some tweaks to it so that 1280x720 looks better.

**Comment by rehsd** [comment]
@DimastyK, I posted an update to https://github.com/rehsd/Emul816or/tree/master/executable. See version 0.14.

**Comment by DimastyK** [comment]
Yes. it works. Thank you! Would be great if you added some examples - PSG and video (like 'rainbow' on screen).
I understand that this is not feasible, but it would be nice if the x86 version was available.

**Comment by rehsd** [comment]
Good to hear! Maybe in December, if I have some spare time, I can gather some sample code from previous projects that would be easy to use. An x86 emulator would be fun to develop. I wish I had more time.  :)
